ably-ios/ARTAuth.h

```diff
@@ -15,7 +15,7 @@
 
 @interface ARTAuthToken : NSObject
```

Comment:
I think we refer to this as `TokenDetails` because it's not actually the token, but instead a type that includes the `token` string and meta data about the token. See https://github.com/ably/ably-java/blob/d0bf20eb1ee9e5011e212fc38e2b09dbee7144db/src/io/ably/rest/Auth.java#L131-L136